lib/dash.coffee

```diff
@@ -55,7 +57,17 @@ plugin = module.exports =
       path = activeEditor.getPath()
       language = activeEditor.getGrammar().name
 
-    spawn('open', ['-g', @createLink(string, path, language)])
+    if platform == "win32"
+      # Exec is used because spawn escapes arguments that contain double-quotes
+      # and replaces them with backslashes. This interferes with the ability to
+      # properly create the child process in windows, since windows will barf
+      # on an ampersand that is not contained in double-quotes.
+      quotedLink = '"' + @createLink(string, path, language) + '"'
+      exec('cmd.exe /c start "" ' + quotedLink, null)
```

Comment:
Thanks! I got most of it, though I haven't verified it. I guess the cruft of it was that I would have assumed that node was handling the argument quoting/escaping/whatever for you so you could do something like `spawn('cmd.exe', ['/c', 'start', '', ... ])` (no double quotes). Sounds like that's not possible then? If it's not, this is already all good to merge anyway :smile: Thanks for explaining a bunch around Windows, I really do appreciate it.